File: Math/Javascript/ReverseNumber.js
function ReverseNumber(number) {
 
    let reverse= 0;

    while (number > 0) {
        let lastDigit = number % 10; // Get the last digit
        reverse = reverse * 10 + lastDigit; // Add it to the reversed number
        number = Math.floor(number / 10); // Remove the last digit
    }

    return  reverse;
}


let input = 1234;
let output =  ReverseNumber(input);
console.log("Number of digits:", output);

File: Math/Java/ReverseNumber.java
import java.util.Scanner;

public class ReverseNumber {
    
     public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Input a number
        System.out.print("Enter a number: ");
        int number = scanner.nextInt();

        int reverse = 0;

      
        while (number> 0) {
            int lastDigit = number % 10;  // Extract the last digit
            reverse = (reverse * 10) + lastDigit; // Add the digit to the reversed number
            number /= 10; // Remove the last digit
        }

        System.out.println("Reversed number: " + reverse);
    }
}
